test/govuk_component/taxonomy_sidebar_test.rb

```diff
@@ -13,53 +13,101 @@ def component_name
 
   test "renders a related items block" do
```

Comment:
This was actually my doing from a previous PR, but could you update this test description to say 'taxonomy sidebar' instead?